store/tikv/gcworker/gc_worker_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/pingcap/tidb/store/mockstore"
 	"github.com/pingcap/tidb/store/tikv"
 	goctx "golang.org/x/net/context"
+	"strconv"
```

Comment:
Put standard packages and third-party packages in different code block.

Reply:
ok